appveyor.yml

```diff
@@ -1,19 +1,41 @@
-version: 1.0.{build}
+version: 1.0-{build}
 
-configuration:
-  - New_Dynarec_Release
-  - Release
-platform:
-  - x86
-  - x64
-
-matrix:
-  exclude:
-    - platform: x64
-      configuration: New_Dynarec_Release
+environment:
+  matrix:
+  - platform: Win32
+    configuration: Release
+  - platform: x64
+    configuration: Release
+  - platform: Win32
+    configuration: New_Dynarec_Release
+  - platform: x64
+    configuration: New_Dynarec_Release
 
 before_build:
-- cmd: git clone --depth 1 https://github.com/mupen64plus/mupen64plus-win32-deps.git ../mupen64plus-win32-deps
-build:
-  project: projects/VisualStudio2013/mupen64plus-core.vcxproj
-  verbosity: minimal
+  - git clone --depth 1 https://github.com/mupen64plus/mupen64plus-win32-deps.git ..\mupen64plus-win32-deps
```

Comment:
Is there any rational about removing `cmd:` ?

Reply:
The `echo` command behaves differently in PowerShell than it usually does in CMD. I'm not really familiar with PS, so I simply use the syntax that is most familiar to me (Batch).

Reply:
I'm not sure to get it. I talk about the `git clone` command. Why removing the `cmd` ?

Reply:
Because it's not necessary. The `git` command is agnostic to the CLI, it has no special behaviours in any specific shell.

Check out other AppVeyor's:
https://github.com/PCSX2/pcsx2/blob/master/appveyor.yml
https://github.com/gonetz/GLideN64/blob/master/appveyor.yml